shared/common-adapters/icon.constants.js

```diff
@@ -750,6 +750,11 @@ const iconMeta_ = {
     extension: 'gif',
     require: require('../images/icons/icon-loader-uploading-16.gif'),
   },
+  'icon-loader-uploading-static-16': {
+    isFont: false,
+    extension: 'gif',
+    require: require('../images/icons/icon-loader-uploading-static-16.gif'),
```

Comment:
guess that's a weird half-attempt at doing this. the code for it never existed and it turns out those actually are animated. i deleted them